BlockEntity/BEFruitpress.cs

```diff
@@ -376,9 +369,11 @@ private bool InteractMashContainer(IPlayer byPlayer, BlockSelection blockSel)
                 var hprops = getJuiceableProps(handStack);
                 if (hprops == null) return false;
 
+                if (hprops.LitresPerItem == null && !handStack.Attributes.HasAttribute("juiceableLitresLeft")) return false;
+
                 var pressedStack = hprops.PressedStack.ResolvedItemstack.Clone();
                 if (MashSlot.Empty) MashSlot.Itemstack = pressedStack;
-                else if (mashStack.StackSize >= 10)
+                else if (juiceableLitresLeft + juiceableLitresTransfered >= juiceableLitresCapacity)

```

Comment:
Now properly displays if the container is full once you've hit juiceableLitresCapacity. Includes transferred liters so it's not possible to add more than a stack's worth of fruit slowly by juicing some and then adding more. Before you could release the press, pull out the mash, insert it back in, and add more fruit until “juiceableLitersLeft” hit 10 again, which would sometimes mean adding a partial fruit worth of juice, allowing for a very tedious way of getting infinite juice out of just one fruit if we didn’t have these checks in place.